From a7333d4a2453d5338c4468a5817248d39a72a9f1 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Thu, 16 Apr 2020 18:03:49 +0800
Subject: [PATCH 1/2] hse: remove the default config for HSE

The NXP HSE config could be added through defconfig clearly.
Avoid the feature is involved by default.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/crypto/hse/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/crypto/hse/Kconfig b/drivers/crypto/hse/Kconfig
index e712669288f6..cd1f4742211c 100644
--- a/drivers/crypto/hse/Kconfig
+++ b/drivers/crypto/hse/Kconfig
@@ -3,7 +3,6 @@
 config CRYPTO_DEV_NXP_HSE
 	tristate "NXP HSE crypto driver"
 	depends on SOC_S32GEN1
-	default m
 	select CRYPTO_HASH
 	select CRYPTO_BLKCIPHER
 	select CRYPTO_AEAD
-- 
2.17.1

